### DISCUSSION TITLE
UE5 Support

### DISCUSSION THREAD

**Discussion by neonzz1** [gildor2/UEViewer#315] [Q&A]
I have a few assets I've brought that I wish to export they all support engine version 5 and up and I've had no luck with them any chance someone has made it work on UE5 projects (This is a personal hobby project that I wish to extract assets from)

**Comment by gildor2** [comment]
Duplicate. https://github.com/gildor2/UEViewer/discussions/263

**Reply by neonzz1** [reply]
Sorry I didn't find the Q&A on that
